packages/wdio-browserstack-service/src/Percy/Percy-Handler.ts

```diff
@@ -0,0 +1,184 @@
+import type { Capabilities } from '@wdio/types'
+import type { BeforeCommandArgs, AfterCommandArgs } from '@wdio/reporter'
+
+import {
+    o11yClassErrorHandler,
+    sleep
+} from '../util.js'
+import PercyCaptureMap from './PercyCaptureMap.js'
+
+import * as PercySDK from './PercySDK.js'
+import { PercyLogger } from './PercyLogger.js'
+
+import { PERCY_DOM_CHANGING_COMMANDS_ENDPOINTS } from '../constants.js'
+
+class _PercyHandler {
+    private _testMetadata: { [key: string]: any } = {}
+    private sessionName?: string
+    private _isAppAutomate?: boolean
+    private isPercyCleanupProcessingUnderway?: boolean = false
```

Comment:
Either prefix all private fields or not, this doesn't seem consistent

Reply:
I would suggest to use `#` for private fields if you are not required to use the in your tests

Reply:
Some of these fields are used in our tests, hence not using `#`

Reply:
Understand, what I am trying to say is to please either prefix private fields with `_` consistently or not at all. Currently it seems very random.

Reply:
Done, prefixed all private fields with _